src/test/java/core/basesyntax/strategy/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.handlerimpl.BalanceOperation;
+import core.basesyntax.strategy.handlerimpl.PurchaseOperation;
+import core.basesyntax.strategy.handlerimpl.ReturnOperation;
+import core.basesyntax.strategy.handlerimpl.SupplyOperation;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+
+public class OperationStrategyImplTest {
+    private static OperationStrategy operationStrategy;
+    private static Map<FruitTransaction.Operation, OperationHandler> operationHandlerMap;
+
+    @BeforeClass
+    public static void setUp() {
```

Comment:
I think you could divide the tests handleImpl into component parts to test each of them. That is, create a separate test for each (BalanceOperation, SupplyOperation, PurchaseOperation, Returnoperation)